packages/gatsby/cache-dir/head/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Helmet } from "react-helmet-async"
+import { useLocation } from "@reach/router"
+
+export const Head = props => {
+  const location = useLocation()
+
+  // This supports passing a gatsby-plugin-image object to the image prop, e.g.
+  // image={getImage(data.metaImage)}
+  const image = props.image?.images?.fallback?.src || props.image
+  const { title, description } = props
+
+  return (
+    <Helmet>
+      {/* General meta tags */}
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content={location.href} />
+
+      {title && (
+        <>
+          <title>{title}</title>
+          <meta name="twitter:title" content={title} />
+          <meta property="og:title" content={title} />
+        </>
+      )}
+
+      {description && (
+        <>
+          <meta name="description" content={description} />
+          <meta name="twitter:description" content={description} />
+          <meta property="og:description" content={description} />
+        </>
+      )}
+
+      {image && (
+        <>
```

Comment:
You're adding `<meta name="description" content={description} />` for the `description`, so for `image` you could also add `<meta name="image" content={image} />`